units.h

```diff
@@ -5,9 +5,13 @@ class Unit {
 private:
 	const std::string m_name;
 	int m_hp;
-	const int m_damage;
+	int m_damage;
+	int m_xp;
+	int m_level;
+	int m_maxHP;
+
 public:
-	Unit(std::string, int, int);
+	Unit(const std::string&, int, int);
 	std::string getName() const { return m_name; }

```

Comment:
👍 viszont akkor már a getName is const referenciát adjon vissza a névre, ott se legyen fölös másolgatás